docs/CONNECTOR_PROTOCOL.md

```diff
@@ -95,10 +95,10 @@ This is our main communication index, used to communicate the connector's config
   last_seen: date;      -> Connector writes check-in date-time
                            regularly (UTC)
   last_sync_error: string;   -> Optional last sync error message
-  last_synced: date;    -> Date/time of last completed sync (UTC)
-  last_indexed_count: number;    -> How many documents were inserted into the index
-  last_deleted_count: number;    -> How many documents were deleted from the index
   last_sync_status: string;  -> last sync Enum, see below
+  last_synced: date;    -> Date/time of last completed sync (UTC)
+  last_indexed_document_count: number;    -> How many documents were inserted into the index
+  last_deleted_document_count: number;    -> How many documents were deleted from the index
```

Comment:
I renamed the field name as I suggested [here](https://github.com/elastic/connectors-ruby/pull/282#discussion_r959697636), to be more consistent.

cc @sphilipse @byronhulcher in case this affects FE